src/inc/corinfo.h

```diff
@@ -2508,6 +2508,12 @@ class ICorStaticInfo
             CORINFO_CLASS_HANDLE        cls
             ) = 0;
 
+    // "System.Int32" ==> CORINFO_TYPE_INT..
+    // "System.UInt32" ==> CORINFO_TYPE_UINT..
+    virtual CorInfoType getTypeForPrimitivNumericClass(
+            CORINFO_CLASS_HANDLE        cls
+            ) = 0;
+
```

Comment:
I don't see where this gets called by the jit ... can you point me at the spot?

Reply:
Also, it has typo in the name.

Reply:
@jkotas Thanks 😅